lib/suspenders/adapters/heroku.rb

```diff
@@ -0,0 +1,52 @@
+module Suspenders
+  module Adapters
+    class Heroku
+      def initialize(app_builder)
+        @app_builder = app_builder
+      end
+
+      def set_heroku_remotes
+        remotes = <<-SHELL
+
+# Set up the staging and production apps.
+#{join_heroku_app('staging')}
+#{join_heroku_app('production')}
+      SHELL
+
+        @app_builder.append_file "bin/setup", remotes
+      end
+
+      def join_heroku_app(environment)
```

Comment:
Not at all, but it was a public method on app builder so I didn't change the visibility. However, we can take this opportunity to do that.